coordinator/src/db/user.rs

```diff
@@ -128,6 +195,8 @@ pub fn login_user(
             fcm_token: token.clone(),
             version: version.clone(),
             last_login,
+            // TODO: this breaks the used referral code
```

Comment:
Break might be the wrong term, but since we allow users to _register_ on login their referral code might not be forwarded. Alternatively we would need to add the referral code on _login_ as well but that's a bit weird. 
So imho, this function here is wrong, we shouldn't create a new user on _login_. :)